packages/Stylers/src/includes.js

```diff
@@ -47,10 +47,40 @@ export const visuallyHidden = css`
   overflow: hidden;
   padding: 0 !important;
   position: absolute !important;
+  white-space: nowrap;
   width: 1px !important;
 `;
 
+export const focusIndicator = css`
```

Comment:
I like this abstraction, but don't understand what they mean or what is their intention, I don't see myself remembering this on top of my head easily, like I do with `stylers.spacer()` aren't the name a little complicated to remember? what about encapsulated this inside of a focus object.

I wouldn't know which one to use if I only see this code. 

export default focusIndicator = {
   inset: ``,
   normal: ``,
   subtle: ``,
   composite: ``,
   compositeInser: ``,
}

Reply:
I'm not so sure it's any easier to remember `focusIndicator.inset` vs `insetFocusIndicator`.  But maybe... since the pattern is somewhat more consistent.  

Alternatively, it could be a function that takes an `options` argument... but that would still probably be about the same cognitive load.

`focusIndicator({ type: 'composite', inset: true })` (with defaults so the most common case could still be `focusIndicator()`. I'm not sure this could be replicated for the Sass version though, so it might have to be two arguments instead of an object.

Anyone prefer this API?

Reply:
Okay, I've taken your feedback into account, and after discussing with Jamie have settled on an API I think I like.  Will update the description above with details.